dpnp/backend/extensions/statistics/common.hpp

```diff
@@ -34,8 +34,30 @@
 // so sycl.hpp must be included before math_utils.hpp
 #include <sycl/sycl.hpp>
 #include "utils/math_utils.hpp"
+#include "utils/type_utils.hpp"
 // clang-format on
 
+namespace dpctl
+{
+namespace tensor
+{
+namespace type_utils
+{
+// Upstream to dpctl
```

Comment:
I have created [dpctl#1900](https://github.com/IntelPython/dpctl/pull/1900) to get rid of that.
So that can be removed once the PR is merged in dpctl and mapped to `dev-milestone` branch.

Reply:
Removed